### ISSUE TITLE
Не з’являється вікно з перекладом деяких текстів персонажів та відсутній переклад деяких тултипів професій

### ISSUE THREAD

**Issue by Wilford1990** [greenya/ClassicUA#39]
Доброго дня! Слава Україні!

Не з’являється вікно з перекладом деяких текстів персонажів. Знайшов я саме наступні (скріни у вкладенні):
1. Gryan Stoutmantle (Гріан Міцнокрив) - при інтерактиві з персонажем, у вікні де постає вибір завдань текст не перекладається (скрін додається);
2. Matias Show (Матіас Гич) - у ланцюгу завдань The Defias Brotherhood (Повстанське Братство) у персонажа та сама ж проблема, що і у Gryan Stoutmantle (Гріан Міцнокрив) (нажаль скрін зробити забув);
3. Те саме було знайдено у приборкувачів грифонів (скрін додається).
4. Також відсутній перекладд у тултипі професії First Aid (Перша(Невідкладна) допомога), в інших професіях на моєму персонажі (Алхімія, Кулінарія, Знайти трави, Риболовля) переклад присутній (скрін додається).

Величезне дякую Вам за даний проєкт, дуже приємно грати в улюблену гру рідною мовою :)

![WoWScrnShot_010524_115230](https://github.com/greenya/ClassicUA/assets/155723803/0bd46991-8cf1-440a-bd61-934d7747b2ff)
![WoWScrnShot_010524_121004](https://github.com/greenya/ClassicUA/assets/155723803/ee4af013-cf6f-4ed3-a61e-ecf9eb4b8bd5)
![WoWScrnShot_010524_121849](https://github.com/greenya/ClassicUA/assets/155723803/2b41d866-81b4-4f4e-835d-6e6f19c8b71e)

**Comment by greenya** [comment]
Так, при взаємодії з нпц, перший діалог де нпц вітає гравця чи повідомляє про якісь локальні чутки й має список взаємодій (нові чи поточні завдання, переліт, банк тощо) -- на жаль наразі не перекладається.

**Comment by Wilford1990** [comment]
> Так, при взаємодії з нпц, перший діалог де нпц вітає гравця чи повідомляє про якісь локальні чутки й має список взаємодій (нові чи поточні завдання, переліт, банк тощо) -- на жаль наразі не перекладається.

З діалогом там де є список взаємодій - зрозуміло (можливість перевести наразі відсутня), та чи може, за можливості, є вірогідність поправити тултип професії "First Aid" (з іншими професіями, які я вивчив на персонажі усе нормально)?

**Comment by Jakanis** [comment]
@Wilford1990 так, не всі професії поки перекладені, працюємо над цим
